lib/wrap.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+'use strict'
+
+const Exclude = require('test-exclude')
+const inspector = require('inspector')
+const {isAbsolute} = require('path')
+const onExit = require('signal-exit')
+const {resolve} = require('path')
+const sw = require('spawn-wrap')
+const uuid = require('uuid')
+const v8ToIstanbul = require('v8-to-istanbul')
+const {writeFileSync} = require('fs')
+
+const argv = JSON.parse(process.env.C8_ARGV)
+
+const exclude = Exclude({
+  include: argv.include,
+  exclude: argv.exclude
+})
+
+;(async function runInstrumented () {
+  try {
+    // bootstrap the inspector before kicking
+    // off the user's code.
+    inspector.open(0, true)
```

Comment:
This isn’t necessary. The in-process Inspector implementation works without launching the WebSocket server, which is what `inspector.open` does.